theano/sandbox/cuda/tests/test_basic_ops.py

```diff
@@ -945,14 +945,14 @@ class TestAlloc(theano.tensor.tests.test_basic.TestAlloc):
     dtype = "float32"
     mode = mode_with_gpu
     shared = staticmethod(cuda.shared_constructor)
-    allocs = [B.GpuAlloc, B.GpuAlloc, tensor.Alloc]
+    allocs = [B.GpuAlloc(), B.GpuAlloc(), tensor.Alloc()]
```

Comment:
Because the future GpuAlloc op will require some parameters to create (namely the context).  It isn't convenient to pass that to the test, so it's simple to have instances than classes.